third_party/move/move-compiler-v2/src/pipeline/avail_copies_analysis.rs

```diff
@@ -217,7 +217,7 @@ impl FunctionTargetProcessor for AvailCopiesAnalysisProcessor {
             return data;
         }
         let target = FunctionTarget::new(func_env, &data);
-        let analysis = AvailCopiesAnalysis::new(target.get_borrowed_locals());
+        let analysis = AvailCopiesAnalysis::new(target.get_pinned_temps(false));
```

Comment:
Search and replace `borrowed` with `pinned` in this file.

Reply:
Done